README.md

```diff
@@ -91,6 +91,10 @@ Using pip from pypi:
 ```
 pip install stable-baselines
 ```
+If you need one of the algorithms depending on MPI, then install using:
```

Comment:
I would put that fist, so people have full installation by default and I would precise what are the algorithms depending on MPI (same for the documentation), otherwise I'm pretty sure we will get issue from people...